src/VisibilityDetectingView.tsx

```diff
@@ -39,49 +37,56 @@ export default class VisibilityDetectingView extends PureComponent<Props> {
 
     // UIManager.measure may not exist during render-testing, which might break the
     // `view.measure` call
-    if (this.view && UIManager.measure) {
+    if (this.view && this.view.measure) {
```

Comment:
suspicious - is this change needed?

Reply:
This was throwing an error saying that it always would resolve to true

Reply:
I see - as the comment mentions above, the test is there for render-testing. I'll see if we can remove and mock instead the function in test env

Reply:
in theory this won't change the behavior since we're testing on the method we're calling, so I'll leave it alone